recipes/jupyterlite-core/meta.yaml

```diff
@@ -0,0 +1,330 @@
+        - {{ pin_subpackage("jupyterlite-core-with-lab", max_pin="x.x.x") }}
+        - {{ pin_subpackage("jupyterlite-core-with-libarchive", max_pin="x.x.x") }}
+        - {{ pin_subpackage("jupyterlite-core-with-mathjax", max_pin="x.x.x") }}
+        - {{ pin_subpackage("jupyterlite-core-with-serve", max_pin="x.x.x") }}
+        - {{ pin_subpackage("jupyterlite-core-with-translation", max_pin="x.x.x") }}
+        - {{ pin_subpackage("jupyterlite-core", max_pin="x.x.x") }}
+        - python >=3.8
+    test:
+      imports:
+        - jupyterlite_core
+      commands:
+        - pip check
+        - jupyter lite --help
+        - pytest -vv --pyargs jupyterlite_core --script-launch-mode=subprocess --cov=jupyterlite_core --cov-report=term-missing:skip-covered --cov-fail-under=82 --no-cov-on-fail
+      requires:
+        - diffoscope  # [linux]
+        - jupyterlite-javascript-kernel
+        - pip
+        - pytest-console-scripts
+        - pytest-cov
+    about:
+      home: https://github.com/jupyterlite/jupyterlite
+      summary: Tools for building JupyterLite sites (with all)
+      doc_url: https://jupyterlite.readthedocs.io
+      license: BSD-3-Clause
+      license_file:
+        - jupyterlite-core/LICENSE
+
+  - name: jupyterlite-javascript-kernel
+    build:
+      noarch: python
+      script: cd jupyterlite-javascript-kernel && {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+    requirements:
+      host:
+        - hatchling >=1.6.3,<2
+        - jupyterlab {{ jupyterlab_version }}
+        - pip
+        - python >=3.8
+      run:
+        - python >=3.8
+        - {{ pin_subpackage("jupyterlite-core", max_pin="x.x.x") }}
+    test:
+      imports:
+        - jupyterlite_javascript_kernel
+      requires:
+        - jupyterlab {{ jupyterlab_version }}
+        - jupyterlite-core
+        - m2-grep  # [win]
+        - pip
+      commands:
+        - pip check
+        - jupyter labextension list
+        - jupyter labextension list 1>labextensions 2>&1
+        - grep -iE "@jupyterlite/javascript-kernel-extension.*v{{ version.replace('.', '\.') }}.*OK" labextensions
+        - jupyter lite build
+        - grep -iE "javascript-kernel-extension" _output/jupyter-lite.json
+    about:
+      home: https://github.com/jupyterlite/jupyterlite
+      summary: JavaScript kernel for JupyterLite running in a Web Worker
+      dev_url: https://github.com/jupyterlite/jupyterlite
+      license: BSD-3-Clause
+      license_file:
+        - jupyterlite-javascript-kernel/LICENSE
+        - jupyterlite-javascript-kernel/jupyterlite_javascript_kernel/labextension/static/third-party-licenses.json
+
+about:
+  home: https://github.com/jupyterlite/jupyterlite
+  summary: Tools for building JupyterLite sites
+  doc_url: https://jupyterlite.readthedocs.io
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  feedstock-name: jupyterlite-core
+  recipe-maintainers:
+    - bollwyvl
+    - jtpio
```

Comment:
For reference: https://github.com/conda-forge/staged-recipes/pull/22344#issuecomment-1498161889